ext/RMagick/rmmain.c

```diff
@@ -123,20 +103,35 @@ static void rm_free(void *ptr)
  *
  * No Ruby usage (internal function)
  */
-static void set_managed_memory(void)
+static inline void managed_memory_enable(VALUE enable)
 {
-    ID enable_mm = rb_intern("RMAGICK_ENABLE_MANAGED_MEMORY");
-
-    if (RTEST(rb_const_defined(rb_cObject, enable_mm)) && RTEST(rb_const_get(rb_cObject, enable_mm)))
+    if (enable)
     {
-        rb_warning("RMagick: %s", "managed memory enabled. This is an experimental feature.");
         SetMagickMemoryMethods(rm_malloc, rm_realloc, rm_free);
-        rb_define_const(Module_Magick, "MANAGED_MEMORY", Qtrue);
     }
-    else
+    rb_define_const(Module_Magick, "MANAGED_MEMORY", enable);
+}
+
+static void set_managed_memory(void)
+{
+    char *disable = getenv("RMAGICK_DISABLE_MANAGED_MEMORY");
+
+    if (disable)
     {
-        rb_define_const(Module_Magick, "MANAGED_MEMORY", Qfalse);
+        managed_memory_enable(Qfalse);
+        return;
     }
+
+#if defined(_WIN32)
+#if defined(IMAGEMAGICK_GREATER_THAN_EQUAL_6_9_0)
```

Comment:
Can you clarify why we don't enable it for versions lower than 6.9.0 on Windows?

Reply:
I tried various things, but it crashes silently when it is old IM for some reason.

https://ci.appveyor.com/project/mockdeep/rmagick/builds/24706171

I gave up to investigate it... sorry😖

Reply:
No problem was just wondering why. Could you add this as a comment in the code or in your squash commit message?

Reply:
Added comment at https://github.com/rmagick/rmagick/pull/697/commits/ec3d796da2d8e530369571c51e9aba76cd8a1520